tests/errors/test_literal.py

```diff
@@ -1,2 +0,0 @@
-def test_literal1():
-    x: i32 = 0123
```

Comment:
Undo the deletion and update tests using `./run_tests.py -u`

Reply:
I am sorry. I did not notice that this test was added with the last PR. I am undoing it. :+1:

Reply:
Remove all the changes in this file.